docs/release-notes/release-notes-0.14.0.md

```diff
@@ -41,6 +41,10 @@ you.
 * [Outdated warning about unsupported pruning was replaced with clarification that LND **does**
   support pruning](https://github.com/lightningnetwork/lnd/pull/5553)
 
+# Tor
+
+* [Improved feedback from malformed address formats](https://github.com/lightningnetwork/lnd/issues/5318).
```

Comment:
nit: incorrect PR number.

Reply:
@guggero Nice catch. Fixed it right now.